src/app/selectors.ts

```diff
@@ -78,7 +78,7 @@ export const superchargeTokenConfigByTokenSelector = (state: RootState) =>
   state.app.superchargeTokenConfigByToken
 
 export const rewardsEnabledSelector = createSelector(
-  [accountAddressSelector, superchargeTokenConfigByTokenSelector],
+  [walletAddressSelector, superchargeTokenConfigByTokenSelector],
```

Comment:
since `accountAddressSelector` was just `mtwAddress ?? walletAddress`, this is only different if `mtwAddress` is non-null but `walletAddress` is null, which should never happen

Reply:
if `mtwAddress` is not null, this value would be `mtwAddress` in the existing code regardless of what `walletAddress` is right? Isn't this changing behavior for user's with `mtwAddress`?

Reply:
if I understand correctly, rewardsEnabledSelector uses the outputs of accountAddressSelector and superchargeTokenConfigByTokenSelector to compute a boolean. My argument is that in all cases we care about, the output of rewardsEnabledSelector will be the same by just using walletAddressSelector in its place here

Reply:
ah, I didn't read the actual implementation of the selector 😁. This sounds good to me.